src/components/instrument-selector/instrument-selector.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import Dropdown, { DropDownOption } from '../dropdown/dropdown'
+import { selectInstrument } from '../../ducks/instruments/actions'
+
+export const InstrumentSelector = ({ selectedInstrumentShortName, selectInstrument }) => (
+  <React.Fragment>
+    <span>Instrument: </span>
+    <Dropdown onChange={event => selectInstrument(event.target.value)} value={selectedInstrumentShortName}>
+      <DropDownOption key='' value="(none)" label="(none)" />
```

Comment:
This option shouldn't exist as it should default to guitar

Reply:
The problem with this, and it also happens with keys, is that if you are viewing ukulele chords and hit f5, it will default to guitar but you will still see the ukulele chords. You would then need to select ukulele and then guitar to see the guitar chords. That's why I added this `(none)` value, so that if it defaults to it you can actually go directly to guitar or ukulele no matter what state you left last time.

Reply:
Oh. It shouldn't because of `value={selectedInstrumentShortName}`. I'll see if I can reproduce it

Reply:
I tried to reproduce with keys dropdown but it works as expected. If I clear `localStorage`, select any key, refresh, the selected key is still selected and the chords in the key are shown. So if you just replicate that for the ukulele dropdown it shouldn't need a `(none)` option.